### ISSUE TITLE
Showing detailed query result.

### ISSUE THREAD

**Issue by erensimsek** [jhipster/generator-jhipster#2790]
Hello everyone,

We added a info that count of query result to our list pages of entities in our project.

May be it can be add to jhipster-generator too without forget i18n support.

You can see with [screenshot here](https://cdn.pbrd.co/images/1iXhOLJt.png).

Example code of screenshot (**companys.html**)

```html
.....
<div class="text-center">
	<div class="info" translate="queryInfo" 
	     translate-values="{page: '{{page}}', queryCount:'{{queryCount}}'}">
		Showing {{((page-1) * 20)==0 ? 1:((page-1) * 20)}} -  
		{{(page * 20) < queryCount ? (page * 20) : queryCount}} 
		of {{queryCount}} items.
	</div>
	<uib-pagination class="pagination-sm" total-items="totalItems" 
	                ng-model="page" ng-change="loadAll()">
	</uib-pagination>
</div>
```

Example code of screenshot (**company.controller.js**)
```javascript
.....
 $scope.loadAll = function() {
            Company.query({page: $scope.page - 1, size: 20, sort: [$scope.predicate + ',' + ($scope.reverse ? 'asc' : 'desc'), 'id']}, function(result, headers) {
                $scope.links = ParseLinks.parse(headers('link'));
                $scope.totalItems = headers('X-Total-Count');
                $scope.queryCount =  $scope.totalItems;
                $scope.companys = result;
            });
        };

...

$scope.search = function () {
            CompanySearch.query({query: $scope.searchQuery}, function(result) {
                $scope.companys = result;
                $scope.queryCount = $scope.companys.length;
            }, function(response) {
                if(response.status === 404) {
                    $scope.loadAll();
                }
            });
        };
```

Example code of screenshot (**global.json**) for i18n support
```json
.....

    "footer": "This is your footer",
    "queryInfo":"Showing {{((page-1) * 20)==0?1:((page-1) * 20)}} - {{(page * 20)< queryCount ? (page * 20) : queryCount}} of {{queryCount}} items.",

....
```

Regards

**Comment by moifort** [comment]
It's a good idea, can you make a PR?

**Comment by erensimsek** [comment]
Of course I can.. Just I need to read submission guidelines.

**Comment by moifort** [comment]
:+1:

**Comment by deepu105** [comment]
:+1: just a suggestion make that to a directive so that you just need to pass as below
```
<jhi-item-count page="page" total="queryCount"></jhi-item-count>
```

**Comment by jdubois** [comment]
Yes that's cool!
As this has been opened for 20 days, are you working on it? Does anyone else wants to do this?
Otherwise we have too much work at the moment, if nobody works on it I will close it.

**Comment by erensimsek** [comment]
I couldn't start work on this cause of personal issues. But I want to fix this. How much time have we to next release?

**Comment by deepu105** [comment]
10 to 15 days may be

Thanks & Regards,
Deepu

**Comment by wmarques** [comment]
I'm working on it !

**Comment by deepu105** [comment]
closed via #3043 @wmarques please add the fix #issue-number keyword in your future PRs for autoclose
